core/actionProxy/owplatform/knative.py

```diff
@@ -239,21 +239,18 @@ def run(self):
                 message['init'] = createInitDataFromEnvironment()
                 dedicated_runtime = True
 
+            preProcessRequest(message)
```

Comment:
good efficiency catch! (again not a typo) ;)

Reply:
This is basically what should have been the last commit, but never got committed before the push.